ShareClient/ShareClient.swift

```diff
@@ -41,22 +47,22 @@ private let maxReauthAttempts = 2
 // ¯\_(ツ)_/¯
 private func dexcomPOST(_ url: URL, JSONData: [String: AnyObject]? = nil, callback: @escaping (Error?, String?) -> Void) {
     var data: Data?
-
+    
```

Comment:
Trailing whitespace. I understand that some people prefer this, but when submitting code to an existing repo, it's a good idea to follow their conventions.